docs/sdk/client-side-sdks/javascript/javascript-openfeature.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Javascript OpenFeature Web Provider
+sidebar_label: OpenFeature
+sidebar_position: 5
+description: How to implement the OpenFeature Web Provider
+sidebar_custom_props: { icon: toggle-off }
+---
+
+# OpenFeature Web Provider
+
+[OpenFeature](https://openfeature.dev/) is an open standard that provides a vendor-agnostic, community-driven API for feature flagging that works with DevCycle.
+
+DevCycle provides a Javascript implementation of the OpenFeature Web Provider interface, if you prefer to use the OpenFeature APIs to interface with DevCycle.
```

Comment:
might want to add a reference to the nodejs one in case you get confused here?

Reply:
I think it is pretty clear that it's for web no?

Reply:
I may be the dumb one - but I equate JS = nodejs all the time when looking for docs

Reply:
the title is `Web Provider`, I think that's pretty explicit